.github/workflows/test.yml

```diff
@@ -1,18 +1,18 @@
 name: Tests
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
 jobs:
   test:
     runs-on: ubuntu-latest
     container:
       # The CKAN version tag of the Solr and Postgres containers should match
       # the one of the container the tests run on.
       # You can switch this base image with a custom image tailored to your project
-      image: openknowledge/ckan-dev:2.9
+      image: openknowledge/ckan-dev:2.10.1
```

Comment:
Can we use `ckan/ckan-dev` instead? Or simply an ubuntu image? We don't know and may not need everything that is installed in this image.

Reply:
but in case `ckan-dev` the latest from the master branch will be pulled. My preference is to have a fixed version rather a dynamic reference.

Reply:
You can easily use `ckan/ckan-dev:2.10.3`.

Reply:
well, 2.10.3 does not exist and I put 2.10.1 to be consistent with what we have in clan-docker. Removed the minor version.